swri_transform_util/nodes/initialize_origin.py

```diff
@@ -90,25 +91,29 @@ def __init__(self):
                                                   local_xy_navsatfix_topic,
                                                   self.navsat_callback, 2)
             self.subscribers = [gps_sub, navsat_sub]
-        else:
-            try:
-                self.get_logger().info("local_xy_origins_param[0]: %lf" % self.local_xy_origins_param.value[0]) 
-                self.get_logger().info("local_xy_origins_param[1]: %lf" % self.local_xy_origins_param.value[1]) 
-                origin_dict={'latitude': self.local_xy_origins_param.value[0],  
-                             'longitude':self.local_xy_origins_param.value[1],  
-                             'altitude':self.local_xy_origins_param.value[2],   
-                             'heading':self.local_xy_origins_param.value[3]}
-            except rclpy.exceptions.ParameterException:
-                message = 'local_xy_origin is "{}", but local_xy_origins is not specified'
-                self.get_logger().fatal(message.format(self.local_xy_origin_param.value))
-                exit(1)
+        elif type(self.local_xy_origins_param.value) == list:
```

Comment:
This should include heading.

Reply:
See my other comment and decide: https://github.com/swri-robotics/marti_common/pull/752#discussion_r1765258907